docs/microbit.rst

```diff
@@ -60,8 +60,10 @@ Classes
 
 .. toctree::
     :maxdepth: 1
-
+    
+    audio
     image.rst
+    sound :ref: `sound`
```

Comment:
Not sure where this reference is going? where is sound defined?

Reply:
Doesn't appear in the output build either:
![image](https://user-images.githubusercontent.com/29712657/100443592-e2ed4300-30a1-11eb-9d24-1a49f514be5f.png)

Reply:
I've found that we can't use `ref`s in the `toctree`, so if we can't to include Sound here as we do with Image, it would probably need it's own page. See https://github.com/bbcmicrobit/micropython/pull/692#discussion_r531716070

Reply:
This thread is outdated as we've found we can create links with this format: Sound <microbit_micropython_api.rst>

As the commit is also outdates and more difficult to find closing this in favour of discussing in https://github.com/bbcmicrobit/micropython/pull/692/files/a1400f846888d43676c305c31a76d8ff723d5227..690c4b9af804c9534c2b5fc447477b880fc9aa20#r533653491